container/innerwindow.go

```diff
@@ -104,6 +130,13 @@ func (w *InnerWindow) SetPadded(pad bool) {
 	w.content.Refresh()
 }
 
+// Title returns the current title of the window
+//
+// Since: 2.6
+func (w *InnerWindow) Title() string {
+	return w.title
+}
```

Comment:
Adding this method in a PR that says it allows changing the button location might become a bit confusing if we ever have to look back on this in the future but I'll let it slide this time. Lets just add a dot at the end of the first comment though.